code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -50,6 +51,7 @@
 
 
 /obj/item/weapon/grenade/chem_grenade/attackby(obj/item/I, mob/user, params)
+	..()

```

Comment:
Proper inheritance; `return ..()` after everything else, and make sure everything here returns and doesn't get far enough to call ..()